### PR TITLE
Split out content-provider reproducer

### PR DESCRIPTION
**Pull request by cjeanner**
It may happen people want to run the content-provider automatically, but
the actual job manually.

It may also happen the content-provider just passes, but the subsequent
"job" run fails - and we of course don't want to re-run the
content-provider.

As a pull request owner and reviewers, I checked that:
- [X] Appropriate testing is done and actually running
- [X] Appropriate documentation exists and/or is up-to-date:
  - [X] README in the role

### CODE REVIEW

File: roles/reproducer/README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ None
 * `cifmw_reproducer_kubecfg`: (String) Path to the CRC kubeconfig file. Defaults to the image_local_dir defined in the cifmw_libvirt_manager_configuration dict.
 * `cifmw_reproducer_repositories`: (List[mapping]) List of repositories you want to synchronize from your local machine to the ansible controller.
 * `cifmw_reproducer_run_job`: (Bool) Run actual CI job. Defaults to `true`.
+* `cifmw_reproducer_run_content_provider`: (Bool) Run content-provider job id needed. Defaults to `true`.
```
Comment (cescgina) [inline roles/reproducer/README.md:13]: typo? `id` should be `if`?

Comment (cjeanner) [reply roles/reproducer/README.md:13]: ah, right.

**Review by cescgina — COMMENTED** [review]
/lgtm

**Comment by pablintino** [comment]
/lgtm
Adding lgtm back

**Pull request merged**